cpp/src/neighbors/mg/snmg.cuh

```diff
@@ -17,7 +17,8 @@
 #pragma once
 
 #include "../detail/knn_merge_parts.cuh"
-#include <raft/core/resource/nccl_clique.hpp>
+#include <raft/core/resource/multi_gpu.hpp>
```

Comment:
this file will go away after merged of [this PR](https://github.com/rapidsai/cuvs/pull/869)